html/images/logos/msl.svg

```diff
@@ -0,0 +1 @@
+<svg id="VOIP" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100"><defs><style>.cls-1{fill:#00a1f4;}.cls-2{fill:#15325f;}</style></defs><title>Icon</title><g id="Mitel_Unified_Communications" data-name="Mitel Unified Communications"><path class="cls-1" d="M59.9,61.5,50,55.8,46.9,54a5,5,0,0,1,0-8L50,44.3l-9.8-5.7a15.1,15.1,0,0,0,0,23l1.7,1.3L50,67.5,77.2,83.2A15.1,15.1,0,0,0,100,70.2V29.8A15.1,15.1,0,0,0,77.2,16.8L50,32.6l8.1,4.7,1.7,1.3,22.4-13a5,5,0,0,1,2.6-.7,5,5,0,0,1,5,5V70.2a5,5,0,0,1-7.6,4.3l-22.4-13"/><path class="cls-2" d="M50,44.2,53.1,46a5,5,0,0,1,0,8L50,55.8l9.9,5.7a15.1,15.1,0,0,0,0-23l-1.7-1.3L50,32.5,22.8,16.8A15.2,15.2,0,0,0,0,29.8V70.2A15.1,15.1,0,0,0,22.8,83.2L50,67.5l-8.2-4.7-1.7-1.3-22.4,13a5,5,0,0,1-7.6-4.4V29.8a5,5,0,0,1,5-5,5,5,0,0,1,2.6.7l22.4,13h0Z"/></g></svg>
```

Comment:
You don't need to duplicate the same file here.  If you logo exists, the icon is used.

Reply:
I thought that was the case, it it wasn't loading the image on the SERVER/devices page. Maybe that was some local issue. I'll remove the duplicate.